protocol/x/clob/keeper/mev.go

```diff
@@ -520,9 +520,15 @@ func (k Keeper) GetMEVDataFromOperations(
 						return nil, err
 					}
 
+					// `insuranceFundDelta` is measured in int64 quote quantums.
+					// It represents up to ~9 trillion USDC which should always be enough for insurance fund delta.
+					// We explicitly panic if there's an int64 overflow.
+					if !insuranceFundDelta.IsInt64() {
+						panic(fmt.Sprintf("insurance fund delta (%v) is not an int64", insuranceFundDelta.String()))
+					}
+
 					mevLiquidationMatch := types.MEVLiquidationMatch{
-						LiquidatedSubaccountId: matchLiquidation.Liquidated,
-						// TODO: Panic if insurance fund delta is not an int64.
+						LiquidatedSubaccountId:          matchLiquidation.Liquidated,
```

Comment:
Mind adding [this ticket](https://linear.app/dydx/issue/CLOB-957/update-mev-calculation-code-to-use-serializableint-for-insurance-fund) as part of a TODO to update this struct to use `SerializeableInt`?